### ISSUE TITLE
`{{debugger}}` in template-only glimmer components has a misleading DX

### ISSUE THREAD

**Issue by mike-north** [emberjs/ember.js#17962]
When using `{{debugger}}` in template-only glimmer components, `context` is understandably `null` because there's no backing object (by design).

The user will see the following upon arriving at a breakpoint
<img width="988" alt="Screen Shot 2019-04-23 at 11 49 51 PM" src="https://user-images.githubusercontent.com/558005/56638274-86eb4e80-6622-11e9-9b13-eae27b6ad517.png">

The following DX issues need a little cleanup

1. `context` no longer applies in this situation
2. the "For example... `context === get('this');`" no longer applies in this situation

We may want to add something about `get('@foo')` to indicate the productive path for debugging these kinds of templates.

It might even be worth splitting this debugging path up (i.e., you'll end up in a template-only-specific `debugger`, or the standard `debugger` ) since, with the introduction of these components, the debugging mechanisms have diverged

```js
  function templateWithBackingObjectDebugCallback(context, get) {
    console.info('Use `context`, and `get(<path>)` to debug this template.'); // for example...

    context === get('this');
    debugger;
  }

  function templateOnlyDebugCallback(get) {
    console.info('Use `get(<@path>)` to debug this template.'); 
    // for example... `get('@foo');`
    debugger;
  }

  function debugCallback(context, get) {
      if (context) templateWithBackingObjectDebugCallback(context, get);
      else templateOnlyDebugCallback(get);
  }


```

**Comment by rwjblue** [comment]
Yep, totally agreed (and the splitting things up example seems like a plausible solution to me).

Going to label this as an octane bug (IMHO bad developer experience **IS** a bug) so we ensure it is addressed before octane is declared finalized.

**Comment by lifeart** [comment]
UP

Is it possible to allow debuger show "args" in this case?

**Comment by olenderhub** [comment]
Please check https://github.com/emberjs/ember.js/pull/21407
